Formula/gambit-scheme.rb

```diff
@@ -0,0 +1,127 @@
+class GambitScheme < Formula
+  desc "Gambit Scheme"
+  homepage "http://gambitscheme.org"
+  url "https://github.com/gambit/gambit.git", :tag => "v4.9.0"
+
+  depends_on "gcc@6"
+
+  def install
+    args = %W[--prefix=#{prefix}]
+
+    ENV["CC"] = Formula["gcc@6"].bin/"gcc-6"
+
+    if build.with? "disable-ssl-verification"
+      system "sed -i -e 's#SSL_CTX_set_default_verify_paths (c->tls_ctx);##g' lib/os_io.c"
+      system "sed -i -e 's#SSL_CTX_set_verify (c->tls_ctx, SSL_VERIFY_PEER, NULL);##g' lib/os_io.c"
+    end
+
+    if build.with? "single-host"
+      args << "--enable-single-host"
+    end
+
+    if build.with? "multiple-versions"
+      args << "--enable-multiple-versions"
+    end
+
+    if build.with? "gerbil-options"
+      args << "--enable-default-runtime-options=f8,-8,t8"
+    end
+
+    if build.with? "poll"
+      args << "--enable-poll"
+    end
+
+    if build.with? "openssl"
+      depends_on "openssl"
```

Comment:
`depends_on` goes at the toplevel

Reply:
except if it's optional.

Reply:
No. `depends_on` never goes inside the `install` block.

Reply:
How do we handle optional dependencies as that was triggering audit.